claraabk/projetoIP
# (hotfix):changed gridsize

## What I did

- Changed tilesize on settings to 24 and width to 16 

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout hotfix/gridsize`
   1. `git pull origin gridsize`
   
1. In VScode open `Game - Settings`

1. Things to do during the code:

   - Open insomnia and postbird to test the requisitions needed

1. Read the changed files section on GitHub

1. In your venv, run the game and check grid size 


**Time spent:** <see your time spent on https://wakatime.com/dashboard >
